concepts/arrays/about.md

```diff
@@ -146,7 +146,7 @@ Read [this article][instanceof-vs-array-is-array] for more information.
 
 ## Array Methods
 
-Some of the [methods][array_methods] that are available on `Array.prototype` can be used to add or remove from the array.
+Some of the [methods][array_methods] that are available on every Array[^0] object can be used to add or remove from the array.
```

Comment:
The word [methods] already links to the page you have in the footnote and footnotes are only used for quotes.

Reply:
ok, removed them!